src/Pagination.xml

```diff
@@ -0,0 +1,65 @@
+<?xml version="1.0" encoding="utf-8"?>
+<widget id="com.mendix.widget.custom.pagination.Pagination" needsEntityContext="false" offlineCapable="true" xmlns="http://www.mendix.com/widget/1.0/">
+    <name>Pagination</name>
+    <description>Add paging to a list view</description>
+    <icon>iVBORw0KGgoAAAANSUhEUgAAAA8AAAAPCAYAAAA71pVKAAAAAXNSR0IArs4c6QAAAZpJREFUKBWFUT1rAkEQnVv3P0gOrPUPCIpiYyekU0gtQvpYCWJhIVhZWGghFoKQxsLKRiz8QPAXWBswQcVCLRQ9k32T3CG3SgbG2X3Om3n7zqB/olAoPAshntxt1+v1041p91wu9/p9J4BLrdsFHI9HQ3FpOBzSfD6nQCBAsViMgGtkt8zVaiWURIpEIpzVapWi0SidTiehkc/ns1ksFuvYZodlWTQej2kymfAADMNmYTfYFSCaR6MRNZtNrpfLhUKhEGWzWer1egTyfr835COZwWCQkI1Gg2un06HpdErhcJjJaomQkJnP5+uYZofC2JzBYEB+v5/whGQyyYke3LfbrZCQCVn9fp/W67XT0O12KZVKkc/n45nxePxtsVg4Hqlv/yXhGrbCwXa7TXgvYrPZUK1W43O5XCav12vNZrMKA38/vBlkkA6HA1ePx0OlUsnpMwyDZTqATYZrICJ3ux1XKSUlEglNpkaGazCg1WoxGTWTydyVqZHhmgpKp9POf7gDd4AHB8M0zRf15l9Lb5rUgI/lcvl+A2nHH7pSDKctgQKWAAAAAElFTkSuQmCC</icon>
+    <properties>
+        <property key="hideUnusedPaging" type="boolean" defaultValue="false">
+            <caption>Hide unused paging</caption>
+            <category>Appearance</category>
+            <description>Hide paging buttons when there is only one page</description>
+        </property>
+        <property key="pagingStyle" type="enumeration" defaultValue="default">
+            <caption>Paging style</caption>
+            <category>Appearance</category>
+            <description>Default rendering as grid</description>
```

Comment:
Default renders like the grid pagination

Reply:
done